src/event_loop.rs

```diff
@@ -263,6 +263,7 @@ impl<H: Handler> EventLoop<H> {
         self.io_process(handler, events);
         self.notify(handler, messages);
         self.timer_process(handler);
+        handler.tick(self);
```

Comment:
I wonder if I should move this above the call to `timer_process`. Here is how the logs look currently:

```
TRACE:mio::event_loop: event loop tick
TRACE:mio::timer: tick_to; now=40; tick=39
TRACE:mio::timer: ticking; curr=Token(18446744073709551615)
TRACE:mio::timer: ticking; curr=Token(18446744073709551615)
TRACE:mob_server::server: Handling end of tick
TRACE:mio::event_loop: event loop tick
TRACE:mio::timer: tick_to; now=42; tick=41
TRACE:mio::timer: ticking; curr=Token(18446744073709551615)
TRACE:mio::timer: ticking; curr=Token(18446744073709551615)
TRACE:mob_server::server: Handling end of tick
```